phpunit/class-gutenberg-rest-block-patterns-controller-test.php

```diff
@@ -10,27 +10,55 @@
  * Unit tests for REST API for Block Patterns.
  *
  * @group restapi
- * @covers Gutenberg_REST_Block_Patterns_Controller
+ * @covers Gutenberg_REST_Block_Patterns_Controller_6_2
+ * @group something
```

Comment:
What's this? :)

Reply:
A really nice leftover of mine, when I was writing the tests :)